### ISSUE TITLE
bugs in Software Layer Plugin

### ISSUE THREAD

**Issue by LinkenMu** [Xilinx/CHaiDNN#121]
After adding my function in Custom class in custom/custom_class.hpp,   and register the function like this below:
```
Custom::Custom()
{
	cout<<"obj create"<<endl;

	custom_func_keys["LRN"]= &Custom::custom_norm;
	custom_func_keys["gemm"]= &Custom::custom_gemm;
	custom_func_keys["identity"]= &Custom::custom_function;
	custom_func_keys["Python"]=&Custom::custom_reorg;
	custom_func_keys["Python_passthrough"]= &Custom::custom_passthrough;
	**custom_func_keys["yolo"]=&Custom::custom_yolo;	//  add this** 
}
```
Then i Build CHaiDNN Software,  copy libparser_arm.so ,libxstack.so to , SD_card/lib /.
Here comes the problem , XportDNN.pyc still can not find yolo. Besides , these are no gemm, identity, Python_passthrough in message below.  These three type are original code.  What 's wrong with this?

> layer_factory.hpp:77] Creating layer yolo-detector
> F1205 17:08:01.489583 30172 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) **Unknown layer type: yolo** (known types: AbsVal, Accuracy, AnnotatedData, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, ConvolutionRistretto, Crop, Data, Deconvolution, DeconvolutionRistretto, DetectionEvaluate, DetectionOutput, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, FcRistretto, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LRNRistretto, LSTM, LSTMUnit, Log, MVN, MemoryData, MultiBoxLoss, MultinomialLogisticLoss, Normalize, PReLU, Parameter, Permute, Pooling, Power, PriorBox, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, VideoData, WindowData)
> 

I find LRN in message above . Then i try to test whether original  custom plugin can be recognized by XportDNN.pyc.  i fail again.
.prototxt is  like this:
```
layer {
    name: "yolo-detector" 
    **type: "LRN"                                  #look here    replace "yolo" to "LRN"**
    bottom: "layer14-conv" 
    top: "yolo_out" 
#    user_defined_layer: true
    xcustom_param { 
	float_args: 80
	float_args: 4 
	float_args: 0.4
	float_args: 0.45 
	float_args: 1.08 
	float_args: 1.19 
	float_args: 3.42 
	float_args: 4.41 
	float_args: 6.63 
	float_args: 11.38 
	float_args: 9.42 
	float_args: 5.11 
	float_args: 16.62 
	float_args: 10.52 

	top_dim: 425
	top_dim: 13
    } 
}
```
run XportDNN.pyc,    message shows error :

> yolo-detector, LRN
> ['layer14-conv_layer14-act_0_split_1'], ['yolo_out']
> Error: Quantization of LRN is not yet supported

Is these a test sample with software layer plugin ?

By the way. The path param after XportDNN.pyc , must be absolute path,  such as  --deploy_model ,--weights,--calibration_directory ...     If  using relative path like doc reference,  it will report can't find xxx.
